tests/test_s3.py

```diff
@@ -69,6 +69,17 @@ def test_list_by_last_modified_date(path):
     assert len(wr.s3.read_json(path, last_modified_begin=begin_utc, last_modified_end=end_utc).index) == 6
 
 
+def test_s3_delete_objects(path):
```

Comment:
Perhaps we can move this to load tests and try with a much higher number of files?